MyCapytain/errors.py

```diff
@@ -15,7 +15,10 @@ class MyCapytainException(BaseException):
 
 
 class JsonLdCollectionMissing(MyCapytainException):
-    """ Error thrown when a JSON LD has now first ressource
+    """ Error thrown when a JSON LD has now first resource
```

Comment:
I think this should read "has no first resource". Sorry I didn't catch this before.